test/types/strands.ts

```diff
@@ -0,0 +1,134 @@
+// Based on https://beta.p5js.org/tutorials/intro-to-p5-strands/
```

Comment:
Minor: for attribution, please add Title and author: "p5.strands: Introduction to Shaders By Luke Plowden" as URL may not be a permalink